plugin/parser/engine/pg/convert.go

```diff
@@ -336,8 +336,11 @@ func convert(node *pgquery.Node, text string) (res ast.Node, err error) {
 
 		if in.InsertStmt.SelectStmt != nil {
 			if selectNode, ok := in.InsertStmt.SelectStmt.Node.(*pgquery.Node_SelectStmt); ok {
-				if insertStmt.Select, err = convertSelectStmt(selectNode.SelectStmt); err != nil {
-					return nil, err
+				// PG parser will parse the value list as a SELECT statement but we not.
```

Comment:
Then what do we treat for the value list?

Reply:
As an Expression or DEFAULT List when we need to use it.

![image](https://user-images.githubusercontent.com/20775801/183853571-4b8a3443-517c-4628-9f1a-bf1f02ccdae1.png)

See https://www.postgresql.org/docs/current/sql-insert.html

Reply:
I think a list of new struct `ValueItemDef` is better than a SELECT statement